docs/versioned_docs/version-v0.17.0/standard_library/cryptographic_primitives/04_ec_primitives.md

```diff
@@ -17,7 +17,7 @@ curve you want to use, which would be specified using any one of the methods
 `std::ec::{tecurve,montcurve,swcurve}::{affine,curvegroup}::new` which take the coefficients in the
 defining equation together with a generator point as parameters. You can find more detail in the
 comments in
-[`noir_stdlib/src/ec.nr`](https://github.com/noir-lang/noir/blob/master/noir_stdlib/src/ec.nr), but
```

Comment:
These links are going to 404 still as there's no `ec.nr` on master. We should point them towards the `ec.nr` file which exists on the v0.17.0 tag and similar for other releases.

Reply:
this is pointing to the `ec` folder on master, but I'll update to the appropriate versions.